release/nlci/nlci_devanagari_winscript/source/nlci_devanagari_winscript.kmn

```diff
@@ -74,12 +74,46 @@ U+200C + 'X' > context beep
 store(WordFinalPunct) ' ",.>!]})' 
 U+200C + any(WordFinalPunct) > index(WordFinalPunct, 2)
 
-                     
 c Add =
 + '=' > '=' dk(eq)
 '=' dk(eq)+ 'r' > 'ऱ'
 '=' dk(eq)+ 'z' > 'ऴ'    
 
+c Restrict the ZW characters 
+U+200C + 'x' > context beep
+U+200D + 'X' > context beep
+U+200C + 'X' > context beep
+U+200D + 'x' > context beep             
+
+c Smart quotes
++ '"' > U+201C
+U+201C + '"' > U+201D
+U+201D + '"' > '"'
+
++ "'" > U+2018
+U+2018 + "'" > U+2019
+U+2019 + "'" > "'"
+
+c Dashes
+'--' + '`' > U+2013 
+U+2013 + '`' > '--' 
+U+2013 + [K_BKSP] > '--'
+'---' + '`' > U+2014
+U+2014 + '`' > '---'
+U+2014 + [K_BKSP] > '---'    
+
+c backtick for change into English
++ '`' > '`' dk(bt)
+'`' dk(bt) + any(allKeys) > index(allKeys, 3)
+any(allKeys) + any(allKeys) > context index(allKeys, 2)
+    
+c numbers
+store(numberKeys) '1234567890'
+store(TamilNumbers) '௧௨௩௪௫௬௭௮௯௦'
```

Comment:
@binilasanki  how come the devanagari keyboard has tamil numbers? could you fix this please?